tests/Model/IndexInfoFunctionalTest.php

```diff
@@ -60,7 +60,9 @@ public function testIsGeoHaystack(): void
         $index = $result->current();
 
         $this->assertEquals($indexName, $index->getName());
-        $this->assertTrue($index->isGeoHaystack());
+        $this->assertDeprecated(function () use ($index): void {
```

Comment:
I forgot this method existed. It dates back to https://github.com/mongodb/mongo-php-library/commit/4b4a7d99d86950adaa5885761fc1894242a07369#diff-abf1fa77a19f720052b75bc802df0260157b9064d8c3c12a80794f01e9acc065R136.

Reply:
I was happy to find it, otherwise I the tests failed.
Symfony has the [`trigger_deprecation`](https://github.com/symfony/deprecation-contracts) and [`expectDeprecation`] for tests (https://symfony.com/doc/current/components/phpunit_bridge.html#write-assertions-about-deprecations)